add_data_to_graph.sh

```diff
@@ -39,9 +39,9 @@ _arg_clear_data="off"
 
 print_help()
 {
-	printf '%s\n' "Upload JSONLD data to a Neurobagel graph"
+	printf '%s\n' "Upload JSONLD and Turtle data to a Neurobagel graph"
 	printf 'Usage: %s [-h|--help] [--(no-)clear-data] <dir> <graph-url> <graph-db> <user> <password>\n' "$0"
-	printf '\t%s\n' "<dir>: Path to directory containing .jsonld files to be uploaded"
+	printf '\t%s\n' "<dir>: Path to directory containing .jsonld and/or .ttl files to be uploaded"
```

Comment:
Not a new thing, but: maybe a note that ALL files will be uploaded that are at this path.